cypress/TestCases/FireboltCertification/DiscoveryLaunchCold.feature

```diff
@@ -115,4 +115,4 @@ Feature: Discovery.launch_ColdLaunch
         Examples:
             | Scenario              | Discovery_Launch_Key                   | Call_Parameters_Initialization_With_Context_Key   | Validation_Key_For_Parameters_Initialization_With_Intent |
             | Passing random string | launch app with null intent source     | get initialization parameters for null intent     | nullintent source for initialization parameters          |
-            | Passing valid string  | launch app with playback intent source | get initialization parameters for playback intent | playbackintent source for initialization parameters      |
+            | Passing valid string  | launch app with playback intent source | get initialization parameters for playback intent | playbackintent source for initialization parameters      |
```

Comment:
There are many feature files which doesn't have any changes related to this ticket. Is it possible to remove those line additions ?  (Unless its a formatting change that was added automatically)

cypress/TestCases/FireboltCertification/DiscoveryLaunchCold.feature
cypress/TestCases/FireboltCertification/DiscoveryLaunchHot.feature
cypress/TestCases/FireboltCertification/Keyboard.feature 
cypress/TestCases/FireboltCertification/LifecycleTestcases/LifecycleBackground.feature

Reply:
it is added automatically. Tried removing them.. bt not fixing